spec/services/vehicles/model_list_service_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Vehicles::ModelListService do
+  let(:service_result) { ['asd', 'ads'] }
+  let(:brand) { Faker::Vehicle.manufacture }
+
+  before do
+    allow_any_instance_of(Vehicles::ModelListService).to receive(:examples).and_return(service_result)
+  end
+
+  subject {
+    Vehicles::ModelListService.perform(brand)
+  }
```

Comment:
Do not use  `{ }` with multiline. Use `do end` or single line. In currect case single line preferable